src/base_model.jl

```diff
@@ -114,7 +114,32 @@ function build_base_model!(ECModel::AbstractEC, optimizer; use_notations=false)
     @variable(model_user,
         0 <= n_us[u=user_set, a=device_names(users_data[u])]
             <= field_component(users_data[u], a, "max_capacity"))
-
+    # Adjusted positive power for single adjustable appliance by user when supplying to public grid
+    @variable(model_user,
+        0 <= P_adj_P_us[u=user_set, e=asset_names(users_data[u], LOAD_ADJ), t=time_set]
+            <= profile_component(users_data[u], e, "max_supply")[t])
+    # Adjusted positive power for single adjustable appliance by user when absorbing from public grid
+    @variable(model_user,
+        0 <= P_adj_N_us[u=user_set, e=asset_names(users_data[u], LOAD_ADJ), t=time_set]
+            <= profile_component(users_data[u], e, "max_withdrawal")[t])
+    # Adjusted energy for single adjustable appliance by user
+    @variable(model_user,
+        profile_component(users_data[u], e, "min_energy")[t] <= 
+            E_adj_us[u=user_set, e=asset_names(users_data[u], LOAD_ADJ), t=time_set]
+            <= profile_component(users_data[u], e, "max_energy")[t])
+    # Energy demand excahge outside the POD by user and appliance
+    @variable(model_user,
+        0 <= ED_adj[u=user_set, e=asset_names(users_data[u], LOAD_ADJ), t=time_set]
+            <= profile_component(users_data[u], e, "energy_exchange")[t])
+    # Fixed power for single fixed appliance by user
+    @variable(model_user,
+        0 <= P_fix_us[u=user_set, f=asset_names(users_data[u], LOAD), t=time_set]
+            <= profile_component(users_data[u], f, "load")[t])
+    # Efficiency of the adjustable load
+    @variable(model_user,

```

Comment:
This variable is not needed, the eta parameter shall be used directly where needed